.github/workflows/benchmark-flow.yml

```diff
@@ -0,0 +1,117 @@
+name: Run a Benchmark Flow
+
+on:
+  workflow_call:
+    inputs:
+      container:
+        type: string
+        default: redisfab/rmbuilder:6.2.7-x64-bullseye
+      module_path:
+        type: string
+        default: bin/linux-x64-release/src/redisgraph.so
+      profile_env:
+        type: number # for default of 0
+      cluster_env:
+        type: string # for default of ""
+      benchmark_glob:
+        type: string
+        default: "*.yml"
+      triggering_env:
+        type: string
+        default: "circleci" # TODO: change to "github-actions" when ready on grafana
+      allowed_envs:
+        type: string
+        default: "oss-standalone"
+      allowed_setups:
```

Comment:
not sure what `allowed_setups` stands for

Reply:
removed. only interesting for cluster+standalone